src/CellData/CellFields.jl

```diff
@@ -368,6 +371,18 @@ function evaluate!(cache,f::CellField,point_to_x::AbstractVector{<:Point})
   collect(point_to_fx)          # Collect into a plain array
 end
 
+# New CellPoint implementation
```

Comment:
This comment is doomed to be outdated, I would remove it. But you can (and should) inform that you have implemented this new procedure in the [changelog file](https://github.com/Balaje/Gridap.jl/blob/eschnett/evaluate/NEWS.md). Take a look at the structure of the file. What we do in a PR is to update the changelog with a summary of the changes being introduced (in the `Unreleased` section). I would add to the changelog that this PR adds (1) support to allow evaluation of FE functions at arbitrary points and (2) the new `CellPoint` implementation.

Reply:
I would also remove the comments within the `CellPoint(xs::AbstractVector{<:Point}, trian::Triangulation, domain_style::PhysicalDomain)`. The code can be understood without them.

Reply:
Hi @ericneiva, I have updated the PR with these changes.